client/src/components/registration/RegistrationForm.js

```diff
@@ -4,6 +4,9 @@ import config from "../../config";
 
 const RegistrationForm = () => {
   const [userPayload, setUserPayload] = useState({
+    username:"",
+    firstName:"",
+    lastName:"",
```

Comment:
these new fields have been marked as `notNullable` in the migration, meaning we need to add them to the front-end validation check in the `validateInput` function below